gcp/modules/gcp-stackdriver-monitoring/resources/alert_policies/compute_instances_insert.json

```diff
@@ -0,0 +1,44 @@
+{
+  "display_name": "Compute Instances Creation Check",
```

Comment:
Consistent with my preferred naming scheme, I might call this one something like "No GCE instances are created".

Reply:
How about "GCE audit log does not contain non-GKE instance creation events" for alert policy name, and "Non-GKE instance creation event found in GCE audit log" for condition name?